Client/Frontend/Library/HistoryPanel.swift

```diff
@@ -522,7 +522,7 @@ class HistoryPanel: SiteTableViewController, LibraryPanel {
         if indexPath.section == Section.additionalHistoryActions.rawValue {
             return []
         }
-        let title: String = .HistoryPanelDelete
+        let title: String = Strings.HistoryPanelDelete
```

Comment:
You can probably leave out the `: String`

Reply:
With BR being weird, I'll fix this in the string cleanup PR. Good point, though. :)